spec/onlyoffice_documentserver_conversion_helper/xml_parser_data_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
If you'll rename this file to `xml_responce_parser_spec.rb` - new `rubocop` error is gone